hkmc2/shared/src/test/mlscript/ctx/CascadingTypeClasses.mls

```diff
@@ -6,87 +6,133 @@ class Bar[B]
 class Baz
 
 
-fun foo[A](using Foo[A], A)
+fun foo[A](using Foo[A], A) = 42
```

Comment:
It is funny that if we don't provide the body, the lowering part will throw an "not in scope" error, which has confused me for a long time (I thought it was a resolver problem)

Reply:
Yes, it's hilarious.